exporter/coralogixexporter/go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/mostynb/go-grpc-compression v1.1.16 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.53.0 // indirect
```

Comment:
You need to add a `replace` directive for this, since it is a module in this repository

Reply:
Fixed. Thank you :)